Pds/.run/docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+version: '3.8'
+
+services:
+
+  database:
+    container_name: pds_database
+    image: mcr.microsoft.com/mssql/server:2022-latest
```

Comment:
I'm not sure that mssql will work on ARM architecture. Is it possible to use instead Azure SQL Edge?

Reply:
probably I don't know something, I thought that docker images should work on every architecture... 
anyway, I updated the image for the database